lib/cookie.js

```diff
@@ -39,7 +39,7 @@ function getExpires (cookieOpts) {
   if (cookieOpts.maxAge) {
     return new Date(Date.now() + cookieOpts.maxAge)
   } else if (cookieOpts.expires) {
-    return cookieOpts.expires
+    return new Date(cookieOpts.expires)
```

Comment:
`expires` should _always_ be a Date. The conversion always happens here.

Reply:
I know that it is not the performance bottleneck, but how about we just store the timestamp instead of the date in the store?

Reply:
Unfortunately, `express-session` stores it as a Date object:
https://github.com/expressjs/session/blob/master/session/cookie.js#L53-L63

So this could be a potential issue if a service using `@fastify/session` and a service using `express-session` share a session store